app/views/generic_object_definition/_custom_button_form.html.haml

```diff
@@ -0,0 +1,11 @@
+- @angular_form = true
+
+= render :partial => "layouts/flash_msg"
+
+%main-custom-button-form-component{"generic-object-defn-record-id" => @generic_object_definition.try(:id),
```

Comment:
Nitpick, but.. the shortening of `definition` to `defn` sounds pretty random here.

I'd rather see `generic-object-definition-id`, `custom-button-group-id` and `custom-button-id` - record is superfluous, and having to rembember to spell it `defn` in just this one place is inconvenient :).

Reply:
Fair enough. I will change it to the full word - `definition `

Reply:
Fixed in b9953e45df7455700bafe312677f6428efec4b3a